src/ipyforcegraph/behaviors/_base.py

```diff
@@ -21,6 +21,9 @@ class Behavior(ForceBase):
 
     _model_name: str = T.Unicode("BehaviorModel").tag(sync=True)
 
+    CONTEXT: Optional[str] = None
```

Comment:
i don't see a big value to defining something new... at present, we _don't_ yet have anything that touches, for example, nodes and links, but certainly could: e.g. `NodeWithOutLinksSelection`. The `rank` probably provides enough.

Reply:
I was using this to explicitly know if a behavior applies to a `node` or a `link` (or to neither).  Otherwise, we have to presume that `Node` or `Link` is in the name of the class...  We could subclass Behavior into `NodeBehavior` and `LinkBehavior`, and have behaviors inherit from there (or from plain `Behavior`)...  The big question this line of thought led me to is that in reality what may have a context is the trait in the Behavior, e.g., you could have a Behavior where one trait applies to `links` and another to `nodes`.  This may eventually pop in with `forces`, although I could not find an example in the forces we have implemented so far, so maybe it's not a bit deal.

Reply:
Yeah, dunno... if it doesn't do antyhing in the as-shipped front end, i don't see a big value in adding the complexity.